### DISCUSSION TITLE
Why are locked paddles and cells do not have borders anymore

### DISCUSSION THREAD

**Discussion by protoknight12** [TodePond/CellPond#260] [General]
Just wondering

**Comment by TodePond** [comment]
Hey :) I explained why I removed locking in my weekly patreon update this week: https://www.patreon.com/posts/unlocked-72371746

Here's a small section from it:

---

![image](https://user-images.githubusercontent.com/15892272/192520131-b76d773a-0c11-4a4b-a01a-cef9dad07406.png)

----

And I'm gradually changing the visual style of the squares to match my usual style - so I removed borders. I will add back in borders for cases where it's needed, but by default there will be no borders

**Reply by protoknight12** [reply]
This is smart
